### PR TITLE
Website Accessibility changes are made

### PR DESCRIPTION
**Pull request by Metaverse-Nabeel**
### Following changes are made in this PR to do the website accessibility.

- Page titles - no issues found
- Image text alternatives**  - no issues found
- Text headings - Changed `ul` at line 227 to `div` 
- Color contrast - no issues found
- Resize - no issues found
- Interaction - @danielamoreno699 - Made aria-label to all the buttons
- Moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - no issues found
- Updated Readme file with the links for the deployed website

### CODE REVIEW

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Metaverse-Nabeel, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- You have done well making your portfolio more accessible 👍 👍 

## Required Changes ♻️
- According to the project requirements, `If issues were found, students should link to the corresponding line of code in the Pull Request change`. Please edit your PR summary and include the links to the changes made in your PR. Please fix this 🙏 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -1,273 +1,339 @@
...
-        <ul class="projectTags">
-          <li class="tag">html</li>
-          <li class="tag">Ruby on rails</li>
-          <li class="tag">css</li>
-          <li class="tag">javascript</li>
-        </ul>
-        <div class="actionBtn">
-          <button class="btn">See Project</button>
+          <div class="actionBtn">
+            <button class="btn" aria-label="See-project">See Project</button>
```
Comment (Metaverse-Nabeel) [inline index.html:106]: Add `aria-label` by @danielamoreno699

File: index.html [hunk 2]
```diff
@@ -1,273 +1,339 @@
...
+            >
+          </div>
+          <ul class="projectTags">
+            <li class="tag">html</li>
+            <li class="tag">Ruby on rails</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
           </ul>
+          <div class="actionBtn">
+            <button class="btn" aria-label="See-project">See Project</button>
```
Comment (Metaverse-Nabeel) [inline index.html:137]: Add `aria-label` by @danielamoreno699

File: index.html [hunk 3]
```diff
@@ -1,273 +1,339 @@
...
+            >
+          </div>
+          <ul class="projectTags">
+            <li class="tag">html</li>
+            <li class="tag">Ruby on rails</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+          </ul>
+          <div class="actionBtn">
+            <button class="btn" aria-label="See-project">See Project</button>
```
Comment (Metaverse-Nabeel) [inline index.html:168]: Add `aria-label` by @danielamoreno699

File: index.html [hunk 4]
```diff
@@ -1,273 +1,339 @@
...
+            >
+          </div>
+          <ul class="projectTags">
+            <li class="tag">html</li>
+            <li class="tag">Ruby on rails</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
           </ul>
+          <div class="actionBtn">
+            <button class="btn" aria-label="See-project">See Project</button>
```
Comment (Metaverse-Nabeel) [inline index.html:198]: Add `aria-label` by @danielamoreno699

File: index.html [hunk 5]
```diff
@@ -1,273 +1,339 @@
...
-        <ul class="projectTags">
-          <li class="tag">html</li>
-          <li class="tag">Ruby on rails</li>
-          <li class="tag">css</li>
-          <li class="tag">javascript</li>
-        </ul>
-        <div class="actionBtn">
-          <button class="btn">See Project</button>
+        <div class="aboutBtn">
+          <button class="resumeBtn" aria-label="Get-resume">Get my resume</button>
```
Comment (Metaverse-Nabeel) [inline index.html:254]: Add `aria-label` by @danielamoreno699

File: index.html [hunk 6]
```diff
@@ -1,273 +1,339 @@
...
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form
+          class="contactMain"
+          action="https://formspree.io/f/xgebklke"
+          method="post"
+        >
+          <label for="user-name" class="labelClass">User name</label>
```
Comment (Metaverse-Nabeel) [inline index.html:309]: Add a label for accessibility and give it class to make its `display: none`

File: index.html [hunk 7]
```diff
@@ -1,273 +1,339 @@
...
+          />
+          <label for="textarea" class="labelClass">Textarea</label>
+          <textarea
+            class="textareaClass"
+            name="user-message"
+            maxlength="500"
+            placeholder="Write your message here..."
+            required
+          ></textarea>
+          <button class="submitBtn" type="submit" aria-label="Submit-form">Get in touch</button>
```
Comment (Metaverse-Nabeel) [inline index.html:334]: Add `aria-label` by @danielamoreno699

File: index.html [hunk 8]
```diff
@@ -1,273 +1,339 @@
...
+          <label for="user-name" class="labelClass">User name</label>
+          <input
+            class="inputClass"
+            type="text"
+            name="user-name"
+            placeholder="Enter name..."
+            maxlength="30"
+            required
+          />
+          <label for="user-email" class="labelClass">User email</label>
```
Comment (Metaverse-Nabeel) [inline index.html:318]: Add a label for accessibility and give it class to make its `display: none`

File: index.html [hunk 9]
```diff
@@ -1,273 +1,339 @@
...
+          <input
+            class="inputClass"
+            type="email"
+            name="user-email"
+            placeholder="Enter e-mail..."
+            required
+          />
+          <label for="textarea" class="labelClass">Textarea</label>
```
Comment (Metaverse-Nabeel) [inline index.html:326]: Add a label for accessibility and give it class to make its `display: none`

File: README.md [hunk 10]
```diff
@@ -51,7 +51,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link] (Not deployed)
+- [Live Demo Link](https://metaverse-nabeel.github.io/Portfolio-Project/)
```
Comment (Metaverse-Nabeel) [inline README.md:54]: In my previous PR this was the optional change required that's why added link to deployed app here.

File: style.css [hunk 11]
```diff
@@ -1079,3 +1079,7 @@
     border: none;
   }
 }
+
+.labelClass {
+  display: none;
+}
```
Comment (Metaverse-Nabeel) [inline style.css:1085]: Added a class to make the display of `label`s to none.

**Review by Metaverse-Nabeel — COMMENTED** [review]
As per accessibility checks following changes are made at the respective lines.

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @Metaverse-Nabeel , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Good communication 
✅ ... Clear description

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**